code/modules/food/food/thecake.dm

```diff
@@ -0,0 +1,298 @@
+		)
+	result = /obj/item/weapon/chaoscake_layer/six
+
+/datum/recipe/chaoscake_layerseven
+	reagents = list("flour" = 120, "milk" = 50, "sugar" = 40, "egg" = 12, "devilskiss" = 200)
+	fruit = list("potato" = 10)
+	items = list(
+			/obj/item/weapon/reagent_containers/food/snacks/dough,
+			/obj/item/weapon/reagent_containers/food/snacks/dough,
+			/obj/item/weapon/reagent_containers/food/snacks/dough
+		)
+	result = /obj/item/weapon/chaoscake_layer/seven
+
+/datum/recipe/chaoscake_layereight
+	reagents = list("flour" = 120, "milk" = 50, "sugar" = 40, "egg" = 12, "cream" = 200)
+	fruit = list("lemon" = 10)
+	items = list(
+			/obj/item/weapon/reagent_containers/food/snacks/dough,
+			/obj/item/weapon/reagent_containers/food/snacks/dough,
+			/obj/item/weapon/reagent_containers/food/snacks/dough
+		)
+	result = /obj/item/weapon/chaoscake_layer/eight
+
+/datum/recipe/chaoscake_layernine
+	reagents = list("water" = 100, "blood" = 100)
+	fruit = list("goldapple" = 50)
+	items = list()
+	result = /obj/item/weapon/chaoscake_layer/nine
+
+/obj/structure/chaoscake
+	name = "An unfinished cake"
+	desc = "A single layer of a strange cake, you can see the cherry paste ooze, but it feels very incomplete..."
+
+	icon = 'icons/obj/food64x64.dmi'
+	icon_state = "chaoscake_unfinished-1"
+	pixel_x = -16
+
+	var/slices = 6
+	var/maxslices = 6
+	var/stage = 1
+	var/maxstages = 9
+	var/edible = 0
+
+	var/regentime = 1000
+	var/interval = 0
+
+	var/static/list/desclist2 = list(
+			"The first layer of a strange cake, you can see the cherry paste ooze.",
+			"The second layer of the cake sits in place now, smelling of pear with delicious colourful cream.",
+			"The third layer of cake adds a strange purple layer, glazed over with frosting. It smells of grapes, but with a hint of something foul underneath.",
+			"With the fourth layer added the cake looks happier again. Reeking of vanilla, it brings up memories of childhood joy.",
+			"The fifth layer is extremely disturbing on that cake. Smelling of pure copper, it seems that bright blood clots are forming on top.",
+			"The cake is getting closer with the sixth layer added, the pink hue smelling of chocolate, with colourful sprinkles on top.",
+			"The first pair of triplets rest on the cake, despite being mostly similar to the first three, an evil aura becomes noticable.",
+			"The second pair of triplets rest on the cake, if you stand on the bright side, you can feel a good aura lifting your mood.",
+			"A chaos cake. Both a creation of dark and light, the two cakes are kept in a careful balance by that mystical coin in the middle. It's said its effects would dissipate if the balance is ever tipped in favour of one side too much, so both sides much be cut equally."
+		)
+
+/*
+/obj/item/weapon/chaoscake_layerone
+	name = "A layer of cake"
+	desc = "a layer of cake, you can see the cherry paste ooze."
+	icon = 'icons/obj/food_ch.dmi'
+	icon_state = "chaoscake_layer-1"
+*/

```

Comment:
You gonna keep this section? Might also want to rename the icon states if not. As well as the names of the layers in the obj path.